app/views/collections/_form.html.erb

```diff
@@ -1,9 +1,19 @@
 <%= simple_form_for @form, html: { class: 'editor' } do |f| %>
   <div id="descriptions_display">
-    <h2 class="non lower">Descriptions</h2>
-    <% f.object.rendered_terms.each do |term| %>
-      <%= render_edit_field_partial term, f: f %>
-    <% end %>
+    <h2 class="non lower">Metadata</h2>
```

Comment:
Ah, ok. It was metadata in work, so I thought metadata here, but if they want Descriptions, that's fine.